src/main/java/com/joona/rest/controller/LibraryController.java

```diff
@@ -0,0 +1,20 @@
+package com.joona.rest.controller;
+
+import net.minidev.json.JSONObject;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LibraryController {
```

Comment:
This is a health check controller, please rename it to HealthCheckController OR LibraryHealthCheckController